central/complianceoperator/v2/scans/datastore/datastore.go

```diff
@@ -0,0 +1,41 @@
+package datastore
+
+import (
+	"context"
+	"testing"
+
+	pgStore "github.com/stackrox/rox/central/complianceoperator/v2/scans/store/postgres"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/postgres"
+)
+
+// DataStore is the entry point for storing/retrieving compliance operator scan objects.
+//
+//go:generate mockgen-wrapper
+type DataStore interface {
+	// GetScanObject retrieves the scan object from the database
+	GetScanObject(ctx context.Context, id string) (*storage.ComplianceOperatorScanV2, bool, error)
```

Comment:
Why ScanObject and not just Scan?

Reply:
went back and forth a bit.  I will probably change it for simplicity.